vars/daosLatestVersion.groovy

```diff
@@ -31,12 +31,16 @@ String distro2repo(String distro) {
 String getLatestVersion(String distro, BigDecimal next_version, String type='stable') {
     String v = null
     String repo = 'daos-stack-daos-' + distro2repo(distro) + '-x86_64-' + type + '-local/'
+    String artifactory_url = env.ARTIFACTORY_URL
+    /* For backwards support, add the 'artifactory' path if it is missing from the env */
+    if (!artifactory_url.endsWith('/artifactory')) {
+        artifactory_url = "${artifactory_url}/artifactory"
+    }
```

Comment:
Should not be needed.   Both Jenkins have been updated to already do this.

Reply:
I figure its better to be safe than sorry.  We can always remove it later.